runtime-sdk/src/modules/accounts/mod.rs

```diff
@@ -857,6 +857,22 @@ impl module::TransactionHandler for Module {
         ctx: &mut C,
         tx: &Transaction,
     ) -> Result<(), modules::core::Error> {
+        // Check whether the transaction is currently valid.
+        let round = ctx.runtime_header().round;
+        if let Some(not_before) = &tx.auth_info.not_before {
+            if round < *not_before {
+                // Too early.
+                return Err(modules::core::Error::ExpiredTransaction);
+            }
+        }
```

Comment:
```suggestion
        if tx.auth_info.not_before.map(|nbf| round < nbf).unwrap_or_default() {
                return Err(modules::core::Error::ExpiredTransaction);
        }
```
I'm being pedantic here, but I do think that the `&` isn't necessary on like 862.